src/error.rs

```diff
@@ -61,6 +61,8 @@ pub enum RoutingError {
     UntrustedMessage,
     #[error(display = "A new SectionInfo is invalid.")]
     InvalidNewSectionInfo,
+    #[error(display = "A Relocation is invalid.")]
+    InvalidRelocation,
```

Comment:
Should this be more precise, i.e., `InvalidRelocationSignature` or something like that? Alternatively, why not just use the `InvalidSignature` variant?

Reply:
This error is similar to `InvalidNewSectionInfo`. Thee problem is if the signature is not correct, the whole Relocation will fail since we will ignore it. So I'm thinking it make sense as it is?

Reply:
OK.